.github/workflows/vulnerabilities.yml

```diff
@@ -0,0 +1,47 @@
+name: Vulnerabilities
+
+on: [pull_request]
+jobs:
+  vulnerabilities:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - name: Rails application check
```

Comment:
If we move this to a shared action we could be explicit about whether the repo we're working with is a Rails app.